patches/chrome-browser-ui-webui-webui_embedding_context.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/ui/webui/webui_embedding_context.cc b/chrome/browser/ui/webui/webui_embedding_context.cc
+index 77ac87a688034c0b144a73065a76010d7397637c..5566c2e3a1194f45e10344fe0231daee3adee1b3 100644
+--- a/chrome/browser/ui/webui/webui_embedding_context.cc
++++ b/chrome/browser/ui/webui/webui_embedding_context.cc
+@@ -113,7 +113,7 @@ class EmbeddingTabTracker {
+   }
+ 
+   // The tab interface currently embedding the host contents.
+-  raw_ptr<tabs::TabInterface> tab_interface_ = nullptr;
```

Comment:
is there a follow-up to this?

Reply:
It ended up being reverted here 36ae4efceb6517da49e17484db7bc74118946c01 and fixed here 97c761cbd272b7d25dd8c51c8ce0d56914a7491b. cc: @simonhong